packages/next/src/client/components/react-dev-overlay/ui/components/errors/error-overlay-layout/error-overlay-layout.tsx

```diff
@@ -119,30 +139,37 @@ export function ErrorOverlayLayout({
             )
           }
         >
-          <DialogContent>
-            <ErrorOverlayDialogHeader>
-              <div
-                className="nextjs__container_errors__error_title"
-                // allow assertion in tests before error rating is implemented
-                data-nextjs-error-code={errorCode}
-              >
-                <span data-nextjs-error-label-group>
-                  <ErrorTypeLabel errorType={errorType} />
-                  {error.environmentName && (
-                    <EnvironmentNameLabel
-                      environmentName={error.environmentName}
-                    />
-                  )}
-                </span>
-                <ErrorOverlayToolbar error={error} debugInfo={debugInfo} />
-              </div>
-              <ErrorMessage errorMessage={errorMessage} />
-            </ErrorOverlayDialogHeader>
+          <Resizer
+            ref={dialogResizerRef}
+            measure={!animating}
+            data-nextjs-dialog-sizer
+          >
+            <DialogContent>
+              <ErrorOverlayDialogHeader>
+                <div
+                  className="nextjs__container_errors__error_title"
+                  // allow assertion in tests before error rating is implemented
+                  data-nextjs-error-code={errorCode}
+                >
+                  <span data-nextjs-error-label-group>
+                    <ErrorTypeLabel errorType={errorType} />
+                    {error.environmentName && (
+                      <EnvironmentNameLabel
+                        environmentName={error.environmentName}
+                      />
+                    )}
+                  </span>
+                  <ErrorOverlayToolbar error={error} debugInfo={debugInfo} />
+                </div>
+                <ErrorMessage errorMessage={errorMessage} />
+              </ErrorOverlayDialogHeader>
+
+              <ErrorOverlayDialogBody>{children}</ErrorOverlayDialogBody>
+            </DialogContent>
+          </Resizer>
 
-            <ErrorOverlayDialogBody>{children}</ErrorOverlayDialogBody>
-          </DialogContent>
           <ErrorOverlayBottomStack
-            errorCount={runtimeErrors?.length ?? 0}
+            errorCount={errorCount}
```

Comment:
Using the same `errorCount` variable as introduced in https://github.com/vercel/next.js/pull/77821